pdo_ibm.c

```diff
@@ -32,7 +32,9 @@
 
 /* If you declare any globals in php_pdo_ibm.h uncomment this:
 */
+#ifdef ZTS
```

Comment:
Same issue here. Should be `#ifdef PASE`, no?

Reply:
Yes you are right given fix.